client/verta/verta/_internal_utils/_artifact_utils.py

```diff
@@ -108,6 +108,8 @@ def ext_from_method(method):
         return 'hdf5'
     elif method in ("joblib", "cloudpickle", "pickle"):
         return 'pkl'
+    elif method == "torch":
+        return "pt"
```

Comment:
Because the Client actually uploads using `cloudpickle` anyway (see thread in #client), let's remove this.
Let me know if it causes a problem.

Reply:
removed